test/metabase/sync/sync_metadata/fields_test.clj

```diff
@@ -14,32 +14,55 @@
    [metabase.util :as u]
    [toucan2.core :as t2]))
 
-(defn- with-test-db-before-and-after-altering
-  "Testing function that performs the following steps:
-   1.  Create a temporary test database & syncs it
-   2.  Executes `(f database)`
-   3.  Drops one of the columns from the test DB & syncs it again
-   4.  Executes `(f database)` a second time
-   5.  Returns a map containing results from both calls to `f` for comparison."
-  {:style/indent [:fn]}
-  [alter-sql f]
-  ;; first, create a new in-memory test DB and add some data to it
+(defn- do-with-test-db [thunk]
   (one-off-dbs/with-blank-db
-    (doseq [statement [ ;; H2 needs that 'guest' user for QP purposes. Set that up
+    (doseq [statement [;; H2 needs that 'guest' user for QP purposes. Set that up
                        "CREATE USER IF NOT EXISTS GUEST PASSWORD 'guest';"
-                       ;; Keep DB open until we say otherwise :)
```

Comment:
I generally enjoy emojis but this smile was causing widespread desctruction to my editor